pypopquiz/popquiz.py

```diff
@@ -13,26 +13,33 @@ def parse_arguments():
     parser = argparse.ArgumentParser(description="pypopquiz interface")
     parser.add_argument("-i", "--input_file", required=True, help="Input JSON file with popquiz info", type=Path)
     parser.add_argument("-o", "--output_dir", required=True, help="Output dir with popquiz data", type=Path)
+    parser.add_argument("-b", "--backend", required=False, help="Backend selection", type=str, default='ffmpeg')
     return vars(parser.parse_args())
 
 
-def popquiz(input_file: Path, output_dir: Path) -> None:
+def popquiz(input_file: Path, output_dir: Path, backend: str) -> None:
     """The main routine, constructing the entire popquiz output"""
 
     input_data = ppq.io.read_input(input_file)
     ppq.io.log("Processing popquiz round {:d}".format(input_data["round"]))
 
+    add_spacer = input_data.get('spacers', False)
+
     for question in input_data["questions"]:
         for source in question["sources"]:
-            ppq.io.get_video(source, output_dir)
+            ppq.io.get_video(source, output_dir, input_file.parent)
 
     q_videos, a_videos = [], []
     for index, question in enumerate(input_data["questions"]):
         question_id = index + 1
         round_id = input_data["round"]
         ppq.io.log("Processing question {:d}: {:s} - {:s}".format(question_id, question["artist"], question["title"]))
-        q_video = ppq.video.create_video("question", round_id, question, question_id, output_dir)
-        a_video = ppq.video.create_video("answer", round_id, question, question_id, output_dir)
+        q_video = ppq.video.create_video(
+            "question", round_id, question, question_id, output_dir, backend=backend, add_spacer=add_spacer
+        )
+        a_video = ppq.video.create_video(
+            "answer", round_id, question, question_id, output_dir, backend=backend, add_spacer=add_spacer
+        )
```

Comment:
My code style is actually to format it like this:
```
        a_video = ppq.video.create_video("answer", round_id, question, question_id, output_dir,
                                         backend=backend, add_spacer=add_spacer)
```
I'm also OK with yours but perhaps it is good to decide here which one we follow. There are no PEP guidelines on this I guess?

Reply:
There is no right answer...I prefer the "indent by 1 always", since it makes the indent of the second line less random (so that if you change the function or variable name, you don't need to re-indent everything). But it still looks bad. I'll change it to your style for now, since that is consistent with the rest of the code.